globus_action_provider_tools/flask/apt_blueprint.py

```diff
@@ -153,6 +179,47 @@ def action_introspect(self):
 
         return jsonify(self.provider_description), 200
 
+    def action_enumerate(self, func: ActionEnumerationType):
+        """
+        Decorates a function to be run as an Action Provider's enumeration
+        endpoint.
+        """
+
+        @functools.wraps(func)
+        def wrapper():
+            if not g.auth_state.check_authorization(
```

Comment:
Can we somehow make use of the implementation in flask_helpers to avoid some repetition? Maybe that requires a bit of re-factor to allow. Maybe not worth it if we ultimately deprecate the non-Blueprint registration method though.